mungegithub/github/github.go

```diff
@@ -471,6 +475,20 @@ func (config *Config) PreExecute() error {
 		config.client.BaseURL = url
 	}
 	config.ResetAPICount()
+
+	// passing an empty username returns information
+	// about the currently authenticated user
+	username := ""
+	user, _, err := config.client.Users.Get(context.Background(), username)
+	if err != nil {
+		return fmt.Errorf("Failed to retrieve currently authenticatd user: %v", err)
+	} else if user == nil {
```

Comment:
Not sure if we can get `nil` for `User` or `User.Login` if we get  no error on the `Get()`, so not clear if these are necessary